html-forms/index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Form</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <div>
+      <label for="email">Email:<br /></label>
```

Comment:
instead of using a br tag, it would be better to wrap the input with a div element to separate the elements on separate lines